Project Red (#999)
dirigeants/klasa: guides/Piece Basics/CreatingExtendables.md

> Extendables are pieces that extend objects or classes by copying its properties and methods like a blueprint.

Reviewer: hmm, this doesn't really make sense
```suggestion
Extendables are pieces that extend objects or classes by adding, editing or removing properties and methods.
```
